bigml/logistic.py

```diff
@@ -283,6 +295,8 @@ def category_probability(self, input_data, unique_terms, category):
         for field_id in input_data :
             coefficients = self.get_coefficients(category, field_id)
             probability += coefficients[0] * input_data[field_id]
+            norm2 += math.pow(input_data[field_id], 2)
```

Comment:
yes, previously all the rest of fields have been transformed and added to the unqiue_terms. I'll change the name to numeric_inputs.